src/sagas/publicChannels/getPublicChannels/getPublicChannels.saga.ts

```diff
@@ -11,15 +10,13 @@ export function* getPublicChannelsSaga(socket: Socket): Generator {
 }
 
 export function* loadPublicChannelsSaga(): Generator {
-  console.log('NECTAR: loadPublicChannelsSaga');
   let channels: IChannelInfo[] = [];
   // while (true) {
-    // console.log('inside nectar while true loop');
-    // yield* put(publicChannelsActions.getPublicChannels());
-    // channels = yield* select(publicChannelsSelectors.publicChannels);
-    // if (channels.length > 0) {
-    //   break;
-    // }
-    // yield* delay(getPublicChannelsDelay);
+  //   yield* put(publicChannelsActions.getPublicChannels());
```

Comment:
Is this saga commented in purpose? Currently is does nothing

Reply:
It breaks new design, will be replaced with new public channels logic in separate commit